frontend/event/eventController.js

```diff
@@ -40,7 +40,8 @@
 
 
         function loadEvents(deferred, getEvents, month, year) {
-            getEvents(actualPage, month, year, eventCtrl.institutionKey).then(function success(response) {
+            getEvents(actualPage, _.get(eventCtrl, 'institutionKey', month),
```

Comment:
it gets very confusing.
You can use named parameters. So you pass an object as a parameter.
So it would be something like 
{actualPage, institutionKey: eventCtrl.institutionKey, month, year}
and in the function assignment you would expect
{actualPage, institutionKey, month, year}